integrations/content.yml

```diff
@@ -27,3 +29,9 @@ features:
   - Vue components in Markdown
   - Full-text search
   - Table of contents generation
+articles:
+  - name: Create a Blog with Nuxt Content
+    link: https://nuxtjs.org/blog/creating-blog-with-nuxt-content
+    author: Debbie O'Brien
+    twitter: https://twitter.com/debs_obrien
```

Comment:
We use handle only for social network links (this way can use for display @usename)

Reply:
ok